### DISCUSSION TITLE
Npmjs: Not able to recover password

### DISCUSSION THREAD

**Discussion by vijay-pawar-aptos-com** [npm/feedback#854] [Registry] [answered]
I have rejoined Aptos company, they have given me same email ID which I used to use 1+ years back. I used same emailID to register in npmjs. As didn't recall  the npmjs password,  I initiated Recover Password I didn't receive any mail to reset the password. tried multiple time too with no result. I also have raised ticket under npm support with Ticket ID: 1972460. But still have not received any response. Pls some one help here.

**Comment by vijay-pawar-aptos-com** [comment, marked as answer]
this issue has been resolved.. my email id was in global suppression list.. once removed, i have started getting npmjs mails.

**Comment by itsuki0927** [comment]
I want to ask you how to delete it in the global blacklist, I also can not receive the corresponding email, is there any operation?

**Reply by MylesBorins** [reply]
please reach out to support
